save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -252,32 +252,29 @@ class LnkUserOrganizationController(
         lnkUserOrganizationService.getSuperOrganizationsWithRole((authentication.details as AuthenticationDetails).id)
     )
 
-    @GetMapping("/by-user")
+    @PostMapping("/by-user-and-filters")
     @RequiresAuthorizationSourceHeader
     @PreAuthorize("permitAll()")
     @Operation(
-        method = "GET",
-        summary = "Get user's organizations by status",
-        description = "Get organizations by status available for the current user.",
+        method = "POST",
+        summary = "Get user's organizations by filters",
+        description = "Get organizations by filters available for the current user.",
     )
     @Parameters(
-        Parameter(name = "status", `in` = ParameterIn.QUERY, description = "this type of organizations", required = false),
+        Parameter(name = "filters", `in` = ParameterIn.DEFAULT, description = "this type of organizations", required = true),
```

Comment:
`this type of organizations` is a bad description for `filters`.

Please pay attention, don't copy and paste code blindly.

Reply:
Done

Reply:
Done